Symfony/CS/Fixer/Contrib/UselessReturnFixer.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author SpacePossum
+ */
+final class UselessReturnFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        for ($index = 0, $count = $tokens->count(); $index < $count; ++$index) {
+            if (!$tokens[$index]->isGivenKind(T_FUNCTION)) {
+                continue;
+            }
+
+            $index = $functionOpen = $tokens->getNextTokenOfKind($index, array('{'));
+            $this->fixFunction($tokens, $functionOpen, $tokens->findBlockEnd(Tokens::BLOCK_TYPE_CURLY_BRACE, $functionOpen));
+        }
+
+        return $tokens->generateCode();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getDescription()
+    {
+        return 'There should not be an empty return statement at the end of a function.';
```

Comment:
I think that would make the fixer `risky`, for example:
```php
<?php
class Test {

    public function myTestFunction($a)
    {
        $b = $a;
        if ($a) {
            $b = include 'includeTest.php';
        }
        return $b;
    }
}

$test = new Test();
$b = $test->myTestFunction(true);
var_dump($b);
die;
```

`includeTest.php`

```php
<?php

if (time() < 0) {
    return 1;
}
return;
```

sure it isn't the most clean code IMHO, but it is valid and the fixer would break it.